cishouseholds/pipeline/high_level_transformations.py

```diff
@@ -592,6 +592,120 @@ def transform_survey_responses_version_digital_delta(df: DataFrame) -> DataFrame
     )
+    df = assign_column_value_from_multiple_column_map(
+        df,
+        "work_status_v1",
+        [
+            [
+                "Employed and currently working",
+                [
+                    "Employed",
+                    "Currently working. This includes if you are on sick or other leave for less than 4 weeks",
+                    None,
+                    None,
+                ],
+            ],
+            [
+                "Employed and currently not working",
+                [
+                    "Employed",
+                    [
+                        "Currently not working -  for example on sick or other leave such as maternity or paternity for longer than 4 weeks",  # noqa: E501
+                        "Or currently not working -  for example on sick or other leave such as maternity or paternity for longer than 4 weeks?",
+                    ],  # noqa: E501
+                    None,
+                    None,
+                ],
+            ],
+            [
+                "Self-employed and currently working",
+                [
+                    "Self-employed",
+                    None,
+                    "Looking for paid work and able to start",
+                    None,
+                ],
+            ],
+            [
+                "Self-employed and currently working",
+                [
+                    "Self-employed",
+                    None,
+                    "Looking for paid work and able to start",
+                    None,
+                ],
+            ],
+            [
+                "Self-employed and currently not working",
+                [
+                    "Self-employed",
+                    None,
+                    "Not looking for paid work. This includes looking after the home or family or not wanting a job or being long-term sick or disabled",
+                    None,
+                ],
+            ],
+            [
+                "Looking for paid work and able to start",
+                [
+                    "Not in paid work. This includes being unemployed or retired or doing voluntary work",
+                    None,
+                    "Looking for paid work and able to start",
+                    None,
+                ],
+            ],
+            [
+                "Not working and not looking for work",
+                [
+                    "Not in paid work. This includes being unemployed or retired or doing voluntary work",
+                    None,
+                    "Not looking for paid work. This includes looking after the home or family or not wanting a job or being long-term sick or disabled",
+                    None,
+                ],
+            ],
+            [
+                "Retired",
+                [
+                    "Not in paid work. This includes being unemployed or retired or doing voluntary work",
+                    None,
+                    "Retired",
```

Comment:
annoyingly we also get this value through from iqvia for retired, "Or retired?" can we add as a list option, same as is done for work_status_v2 map?